README.md

```diff
@@ -49,6 +50,7 @@ this.repository = entityManager.getRepository('users');
       actions.bind="actions"
       populate="user"
       footer.bind="footer"
+      details-view="./details"
```

Comment:
https://github.com/VMBindraban/aurelia-datatable/blob/971340722a6c21f4cb82d7ee79cfc3a2f1340e84/README.md

Also in the `doc/usage.md`